interface/frontend/build/webpack.base.conf.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     extensions: ['.js', '.vue', '.json'],
     alias: {
       'vue$': 'vue/dist/vue.esm.js',
-      '@': resolve('src'),
+      '@': resolve('src')
```

Comment:
Please don't make changes not mentioned in the commit message :)

Reply:
Ok, I will keep that in mind for next time.